code/controllers/subsystems/vote.dm

```diff
@@ -255,7 +255,7 @@ SUBSYSTEM_DEF(vote)
 
 		to_world("<font color='purple'><b>[text]</b>\nType <b>vote</b> or click <a href='?src=\ref[src]'>here</a> to place your votes.\nYou have [config.vote_period / 10] seconds to vote.</font>")
 		if(vote_type == VOTE_CREW_TRANSFER || vote_type == VOTE_GAMEMODE || vote_type == VOTE_CUSTOM)
-			world << sound('sound/ambience/alarm4.ogg', repeat = 0, wait = 0, volume = 50, channel = 3)
+			world << sound('sound/ambience/matteralarm.ogg', repeat = 0, wait = 0, volume = 50, channel = 3)

```

Comment:
```suggestion
			world << sound('sound/misc/notice1.ogg', repeat = 0, wait = 0, volume = 50, channel = 3)

```

matteralarm is too annoying and long (9 secs) better to just go with what we had a while back aka the red alert sound.